README.md

```diff
@@ -115,21 +115,21 @@ make -j4
 
 This should show the following compilation message on the screen:
 ```
-g++ -O -W -pedantic-errors -Wextra -Werror -Wformat -Wformat-nonliteral -Wpointer-arith -Wcast-align -Wconversion  -g  -std=c++98 -MMD -MF src/xmlparser/MemSpecParser.d -iquote src -o src/xmlparser/MemSpecParser.o -c src/xmlparser/MemSpecParser.cc
```

Comment:
In the next iteration of the readme we should really get rid of this list...it's fairly pointless :)